base/test.jl

```diff
@@ -11,6 +11,8 @@ test set that throws on the first failure. Users can choose to wrap
 their tests in (possibly nested) test sets that will store results
 and summarize them at the end of the test set with `@testset`.
 """
+:Test # cf. #22288
```

Comment:
I would say all commits can be squashed on merge, and I can squash before if prefered.